ui/cap-react/src/antd/admin/formComponents/SchemaTreeItem.js

```diff
@@ -63,7 +63,7 @@ const SchemaTreeItem = ({
           uiSchema["ui:field"] === "idFetcher" &&
           uiSchema["ui:servicesList"].length < 3
         ) {
-          type = uiSchema["ui:servicesList"][0].value;
+          type = uiSchema["ui:servicesList"][0];
```

Comment:
- make a bit safer, if array is empty?
- maybe add `unknown` as type, so we display a diff icon (e.g. a questionmark icon `?`)

Reply:
Unless you manually change it in the schema it is not going to be empty since through the UI once only one option is left selected (e.g. Zenodo) the field turns into the specific field (e.g. a Zenodo field). But in any case actually this `if` is not even necessary, since all those idGetter fields share the same icon (which is already chosen based on `ui:field: "idGetter"`). Anyway I made it a bit safer with the `unknown` base type, but in case it shows up it'll mean something is wrong.